SalesforceSDK/Salesforce.SDK.Core/Source/Net/HttpCall.cs

```diff
@@ -89,15 +91,20 @@ public static string MimeType(this ContentTypeValues contentType)
     /// </summary>
     public class HttpCall
     {
-        private readonly ContentTypeValues ContentType;
-        private readonly HttpCallHeaders Headers;
-        private readonly HttpMethod Method;
-        private readonly string RequestBody;
-        private readonly string Url;
-        private readonly HttpClient WebClient;
-        private Exception HttpCallErrorException;
-        private string ResponseBodyText;
-        private HttpStatusCode StatusCodeValue;
+        /// <summary>
+        /// Use this property to retrieve the user agent.
+        /// </summary>
+        public static string UserAgentHeader { private set; get; }
+        private const string UserAgentHeaderFormat = "SalesforceMobileSDK/3.1 {0}";
```

Comment:
Where is the rest of it (see https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/master/libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java#L911)